cmd/axelard/cmd/vald/evm/evm.go

```diff
@@ -507,7 +507,9 @@ func parseGatewayTxConfirmationParams(cdc *codec.LegacyAmino, attributes map[str
 	err error,
 ) {
 	parsers := []*parse.AttributeParser{
-		{Key: evmTypes.AttributeKeyChain, Map: parse.IdentityMap},
+		{Key: evmTypes.AttributeKeyChain, Map: func(s string) (interface{}, error) {
```

Comment:
For consistency, also change the type for chain in `parseNewChainParams`.

Reply:
Interesting. That one is useless at this point cuz we never do confirm new chain anymore. I'll delete it

Reply:
Not a consistency issue, it would actually break if we keep it that way. Good catch.